lib/engine/config/game/g_18_chesapeake.rb

```diff
@@ -691,7 +697,8 @@ module G18Chesapeake
       "tiles": [
         "yellow",
         "green",
-        "brown"
+        "brown",
+        "gray"
```

Comment:
This was a bug pointed out in chat, doesn't look fixed yet in the code so added here.